python/pandemic_simulator/script_helpers/locations.py

```diff
@@ -5,11 +5,46 @@
 import numpy as np
 
 from ..environment import Home, Location, CityRegistry, GroceryStore, Road, Cemetery, Hospital, \
-    Office, School, SimTimeTuple, HospitalState, ContactRate, BusinessLocationState, \
-    NonEssentialBusinessLocationState, RetailStore, BarberShop, PopulationParams
+    Office, School, Restaurant, SimTimeTuple, HospitalState, ContactRate, BusinessLocationState, \
+    NonEssentialBusinessLocationState, RetailStore, BarberShop, PopulationParams, Bar
 
 __all__ = ['make_standard_locations']
 
+# helper method that encapsulates the addition of bars
+def add_bars(all_locs, location_type_to_params,
+            registry: CityRegistry,
+            road):
+    if Bar in location_type_to_params:
+        all_locs += [Bar(
+            registry=registry,
+            name=f'bar_{i}',
+            road_id=road.id,
+            init_state=NonEssentialBusinessLocationState(
+                is_open=True,
+                contact_rate=ContactRate(1, 1, 0, 0.7, 0.6, 0.7),
+                visitor_capacity=location_type_to_params[Bar].visitor_capacity,
+                open_time=SimTimeTuple(hours= \
+                    tuple([x for x in range(0,24) if x <= 2 or x >= 21]), week_days=tuple(range(1, 7)))),
+            numpy_rng=numpy_rng
+        ) for i in range(location_type_to_params[Bar].num)]
+
+# helper method
+def add_restaurants(all_locs, location_type_to_params,
+            registry: CityRegistry,
+            road):
+    if Restaurant in location_type_to_params:
+        all_locs += [Restaurant(
+            registry=registry,
+            name=f'restaurant_{i}',
+            road_id=road.id,
+            init_state=BusinessLocationState(
+                is_open=True,
+                contact_rate=ContactRate(1, 1, 0, .3, .35, .35),
+                visitor_capacity=location_type_to_params[Restaurant].visitor_capacity,
+                open_time=SimTimeTuple(hours=(12, 1, 20, 21, 22), week_days=tuple(range(1, 7)))),
```

Comment:
The `open_times` are same for stage 0 and all stages (except for 4). So why not simply use `lock` regulation?